launchpod/angular-launchpod/src/app/login-popup/login-popup.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-login-popup',
+  templateUrl: './login-popup.component.html',
+  styleUrls: ['./login-popup.component.css']
+})
+export class LoginPopupComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
```

Comment:
Nit: extra space and let's keep {} on the same line

Reply:
done.